AnkiDroid/src/main/java/com/ichi2/anki/DeckPickerFloatingActionMenu.kt

```diff
@@ -64,6 +66,9 @@ class DeckPickerFloatingActionMenu(
         mLinearLayout.alpha = 0.5f
         mStudyOptionsFrame?.let { it.alpha = 0.5f }
         isFABOpen = true
+        if (deckPicker.baseSnackBar != null && deckPicker.baseSnackBar?.isShown == true)deckPicker.baseSnackBar?.dismiss()
+        if (createDeckSnackBar != null && createDeckSnackBar?.isShown == true)createDeckSnackBar?.dismiss()
+
```

Comment:
* `deckPicker.baseSnackBar?.isShown == true` handles the case where `deckPicker.baseSnackBar == null`
* add a space after the `)` and the follow-on

```suggestion
        // ensure the snackbar doesn't obscure the menu
        if (deckPicker.baseSnackBar?.isShown == true) deckPicker.baseSnackBar?.dismiss()
        if (createDeckSnackBar?.isShown == true) createDeckSnackBar?.dismiss()
```

Reply:
done